test/__snapshots__/appsync.test.ts.snap

```diff
@@ -7,72 +7,72 @@ Array [
   #return($context.stash.return__val)
 #end
 {
-  \\"version\\": \\"2018-05-29\\",
-  \\"method\\": \\"POST\\",
-  \\"resourcePath\\": \\"/\\",
-  \\"params\\": {
-    \\"headers\\": {
-      \\"content-type\\": \\"application/x-amz-json-1.0\\",
-      \\"x-amz-target\\": \\"AWSStepFunctions.DescribeExecution\\"
-    },
-    \\"body\\": {
-      \\"executionArn\\": $util.toJson('exec1')
-    }
+\\"version\\": \\"2018-05-29\\",
+\\"method\\": \\"POST\\",
+\\"resourcePath\\": \\"/\\",
+\\"params\\": {
+  \\"headers\\": {
+    \\"content-type\\": \\"application/x-amz-json-1.0\\",
+    \\"x-amz-target\\": \\"AWSStepFunctions.DescribeExecution\\"
+  },
+  \\"body\\": {
+    \\"executionArn\\": $util.toJson('exec1')
```

Comment:
Why did this change

Reply:
Looks like it just tabbed backwards
![image](https://user-images.githubusercontent.com/289213/173890264-f27dd5a1-e4ce-472c-be34-9374c8fd3e93.png)

Ahh, found it : https://github.com/functionless/functionless/pull/249/files#diff-961ed18b565d502c0f3e3673413b522c59ba0121e9bec07aca2807cf73309addL1396-R1116